modules/react-native-login/index.js

```diff
@@ -1,4 +1,6 @@
-import React from "react";
+import React, { useContext } from "react";
+import { OptionsContext } from "@options";
+import AppMenu from "../app-menu";
```

Comment:
Was this used for testing? It should be removed

Reply:
Yeah, Good Catch.